src/core/adapters/notion.ts

```diff
@@ -86,7 +92,10 @@ async function scan(url: URL): Promise<TicketData[]> {
     .post("api/v3/getRecordValues", request)
     .json<NotionTicketResponse>();
 
-  return getTickets(response, id);
+  return getTickets(response, id).map((ticket) => ({
+    url: buildUrl(url, slugId),
```

Comment:
As you mentioned yesterday: How about taking the URL and only setting the path to the `slugId`. So we end up with something like `https://www.notion.so/8c2499ce59654658a1037c11bdd50523` which will be redirected accordingly by notion itself.

Does this fix the problem for non-organization users? I forgot the details about it, sorry 🙈

Reply:
> Does this fix the problem for non-organization users? I forgot the details about it, sorry 🙈

Yes! Will add a commit to change this.